app/controllers/Utils.scala

```diff
@@ -13,12 +13,14 @@ object Utils {
   /**
    * Return base url given a request. This will add http or https to the front, for example
    * https://localhost:9443 will be returned if it is using https.
+   *
+   * Change type from Request[Any]
    */
-  def baseUrl(request: Request[Any], absolute: Boolean = true) = {
+  def baseUrl(request: RequestHeader, absolute: Boolean = true) = {  
```

Comment:
Can you explain this change? Thank you.

Reply:
views/dataset.scala.html's has the request, that works to get the needed baseURL
@Html(Json.prettyPrint(dataset.to_jsonld(Utils.baseUrl(request))))
That page already has too many args, making sending the other request type not easy,
&I tried a 2nd version of baseUrl, but was suggested that we just keep one method